Dockerfile.test

```diff
@@ -16,18 +18,21 @@ RUN wget http://python.org/ftp/python/$pyVer/Python-${pyVer}.tgz && \
 	make && \
 	checkinstall
 
-RUN apt-get install -y python-dev python-pip
-RUN pip install psutil && \
-    pip install netifaces && \
-	pip install bottle && \
-	pip install requests && \
-	pip install python-dateutil
-
 RUN mkdir /crawler
 COPY . /crawler/
 WORKDIR /crawler/
 
-RUN pip2 install --no-index --find-links="packaging/python_packages/" psutil bottle requests pydoubles netifaces kafka-python docker-py pykafka
+RUN apt-get install -y python-dev python-pip
```

Comment:
is python 2.7.5 such a hard dependency that we can't just install python with apt-get?